src/tools/pontoon/resources.md

```diff
@@ -2,35 +2,43 @@
 
 <!-- toc -->
 
-Pontoon features built-in access to different translation memories. These resources are available in the translation workspace and, for machinery, on a [standalone web page](https://pontoon.mozilla.org/machinery/).
+Pontoon features built-in access to different translation memories. These resources are available in the translation workspace. Machinery may also be accessed via the navigation bar as a standalone web page.
 
-## History tab
+## History
 
-The history tab shows all of the suggestions and translations that have been submitted for the current source string.
-
-In the tab, the number of available entries is visible alongside the `HISTORY` title. The history entries are located below the tab.
+Pontoon displays the translation history for strings below the editor. The current approved translation for a source string appears at the top and all previous suggestions and translations beneath.
```

Comment:
We also explain this (in slightly more detail) in line 29. Can we bring everything here?

Reply:
This part is laying out how the editor/translation history appear in Pontoon, while line 29 covers how the DIFF function works. I think the current order makes sense to introduce parts of translation history before going into detail on DIFF.